docusaurus/extensions/catalog/fleet-workspace-ui.yaml

```diff
@@ -0,0 +1,6 @@
+name: Fleet workspace members UI
+author: Negashev
+authorUrl: https://negash.ru
+description: Management users for fleet workspace
```

Comment:
This looks like gorizond specific permissions based around gorizond annotations on the workfleet resource. The description should reflect this is a permissions based system based on that (rather than native kube or rancher)